pkg/plugins/platform/local.go

```diff
@@ -427,3 +425,24 @@ func (l *LocalApp) AddHostsEntry() error {
 	err = system.RunCommandPipe("sudo", hostnameArgs)
 	return err
 }
+
+// ContainerWait provides a wait loop to check for container in "healthy" status.
+func (l *LocalApp) ContainerWait(timeout time.Duration, containerType string) error {
```

Comment:
Don't we want this in a more general location? Seems like a candidate for pkg/util/docker.go (in https://github.com/drud/ddev/pull/146